wikimedia/wikipedia-ios
# Talk page new topic error

**Phabricator:** https://phabricator.wikimedia.org/T312314

### Notes
* This PR is based on https://github.com/wikimedia/wikipedia-ios/pull/4367, do not merge before 
* We're adding some more error banners when posting topics and when some menu actions fail
* There's an alert view adjustment made specifically for the `TalkPageTopicComposeViewController` since it was causing the alert to display incorrectly

### Test Steps
1. Go to a talk page, create a new topic, and with the internet connection interrupted, try to post
2. A banner should display on the top of the screen
3. Do the same with the menu actions, this will display an alert
